tensorflow/core/kernels/dml_matrix_diag_ops.cc

```diff
@@ -184,9 +184,12 @@ class DmlMatrixDiagKernel : public DmlKernel {
       batch_size *= input_shape.dim_size(i);
     }
 
-    auto elem_count_per_batch = static_cast<uint32_t>(input_shape.num_elements() / batch_size);
-    auto output_height = static_cast<uint32_t>(output_shape.dim_size(output_shape.dims() - 2));
-    auto output_width = static_cast<uint32_t>(output_shape.dim_size(output_shape.dims() - 1));
+    auto elem_count_per_batch =
```

Comment:
[nit] Why though? 🙃 It's more readable on single lines rather than interleaved names and equations.

![image](https://user-images.githubusercontent.com/1809166/118747290-97c11b00-b80e-11eb-9bc1-01fb0dbdaaa3.png)

(resolve if this is stupid clangformat related)

Reply:
This is stupid clangformat :(